.github/workflows/build.yaml

```diff
@@ -3,10 +3,10 @@ name: build
 on:
   push:
     branches:
-      - kirkstone
+      - scarthgap
   workflow_dispatch:
   pull_request:
-    types: [ labeled ]
+    types: [labeled]
```

Comment:
Seems like there is a yaml auto formatter in effect here...I would avoid the auto formatting stuff for now, and we can format all of the yaml files in a follow up PR (and do it to both kirkstone and scarthgap branches).